### PR TITLE
PRO-161 Task: Android Dependencies Update

### PR DESCRIPTION
**Pull request by divyaBacancy**
jcenter to maven central migration and update some dependencies.

### CODE REVIEW

**Comment by divyaBacancy** [comment]
> 🤞 fingers crossed!

Good Luck!

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just one small thing - other than that, LGTM

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,12 +25,13 @@
...
-            android:name=".MainApplication"
-            android:allowBackup="false"
-            android:icon="@mipmap/ic_launcher"
-            android:label="@string/app_name"
-            android:theme="@style/AppTheme">
+        android:name=".MainApplication"
+        android:allowBackup="false"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:fullBackupContent="true"
```
[the remark this answers is not in the document]
Comment (divyaBacancy) [reply android/app/src/main/AndroidManifest.xml:32]: allowBackup false means application data can not be backed up and restored by the user using adb backup and adb restore. As per Ardian suggestion we had set android:allowBackup false.

[the remark this answers is not in the document]
Comment (divyaBacancy) [reply android/app/src/main/AndroidManifest.xml:32]: android:fullBackupContent attribute points to an XML file that contains full backup rules for [Auto Backup](https://developer.android.com/guide/topics/data/autobackup). These rules determine what files get backed up.

android:fullBackupContent is mandatory for com.instabug.library:instabug-core:10.13.0
For more details: https://developer.android.com/guide/topics/manifest/application-element

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**